utils/email.js

```diff
@@ -15,7 +15,7 @@ const sendMail = async (options) => {
             ...options
         });
     } else {
-        console.log('sendMail:', options);
+        emitEmailSentEvent(options);
```

Comment:
let's still make sure to log to console as it is convenient for debugging locally in `development` environment (maybe already wired elsewhere but I didn't notice?)

Reply:
Yup, echoing this data to console is still supported; check out `test/2fa.test.js:34` and `test/app.test.js:43` -- now you can turn on echoing either security codes, or message contents, or both, from the top of the test file directly optionally if you need the extra verbosity, but during normal runs the tests don't create extra console.log output :+1:  

It's worth noting that the output is also formatted for easy readability now (no \n's visible; they are actually converted to newlines) when it _is_ console.logged during a test run; default behavior logs everything that is not HTML by default -- acceptance tests coming right after this PR give us visibility into HTML output in a much nicer way than console.logs